modules/os2forms_encrypt/src/Form/SettingsForm.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\os2forms_encrypt\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Link;
+
+/**
+ * Class Os2FormsEncryptAdminForm.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * The name of the configuration setting.
+   *
+   * @var string
+   */
+  public static string $configName = 'os2forms_encrypt.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames(): array {
+    return [self::$configName];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'os2forms_encrypt_admin_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('os2forms_encrypt.settings');
+
+    $link = Link::createFromRoute($this->t('administration'), 'entity.key.collection');
+    $form['notice'] = [
+      '#type' => 'inline_template',
+      '#template' => '<h3>{{ title }}</h3><p>{{ message|t }}</p><p>{{ adminMessage|t }}</p>',
+      '#context' => [
+        'title' => 'Please note',
+        'message' => 'The encryption key that comes with this module should <strong>not</strong> be used and should be changed before encrypting anything.',
+        'adminMessage' => 'You can modify the key (named "webform") in the keys ' . $link->toString() . ' panel. Additionally, the execution of this command can generate a new 256-bit key for you: <pre>dd if=/dev/urandom bs=32 count=1 | base64 -i -</pre>',
```

Comment:
Should we allow translation of these?

Reply:
They are translated in the inline template above where they er twig pipe into `t` :-)

Reply:
Oh, i missed that